src/url-helper.ts

```diff
@@ -28,19 +28,19 @@ export function getServerUrl(url?: string): URL {
   return new URL(urlValue)
 }
 
-export function getServerApiUrl(url?: string): string {
-  let apiUrl = 'https://api.github.com'
-
-  if (isGhes(url)) {
-    const serverUrl = getServerUrl(url)
-    apiUrl = new URL(`${serverUrl.origin}/api/v3`).toString()
-  }
-
-  return apiUrl
+export function getServerApiUrl(): string {
+  return process.env['GITHUB_API_URL'] || 'https://api.github.com'
 }
 
-export function isGhes(url?: string): boolean {
-  const ghUrl = getServerUrl(url)
+export function isGhes(): boolean {
```

Comment:
This logic now matches what we have in [actions-toolkit](https://github.com/actions/toolkit/blob/ee93b05ee9979c0f7c22e6b648d9fb11b024eeb2/packages/artifact/src/internal/shared/config.ts#L26).